test/integration/google_sso_test.rb

```diff
@@ -12,7 +12,7 @@ class GoogleSSOTest < ActionDispatch::IntegrationTest
       wait_for_element 'Sign in with Google'
       click_link 'Sign in with Google'
 
-      assert has_content? 'Sign out'
+      assert has_content? "#{@user.name}"
```

Comment:
It'll work either way, but you can just do `@user.name` here without the quotes and interpolation! @user.name is already a string.